production/bin_aws/start.sh

```diff
@@ -17,6 +17,20 @@ function config_replace() {
 	sed -i'.bak' -e ${CMD} ${FILE}
     fi
 }
+function getParsedGithubToken(){
```

Comment:
Please use consistent spacing (e.g. between functions and between tokens) and consistent tabs.